utils/src/test/kotlin/DeclaredLicenseProcessorTest.kt

```diff
@@ -23,8 +23,8 @@ import com.here.ort.spdx.SpdxExpression
 import com.here.ort.spdx.SpdxLicenseIdExpression
 import com.here.ort.spdx.SpdxLicenseIdMapping
 import com.here.ort.spdx.SpdxLicenseStringMapping
-import io.kotlintest.shouldBe
 
+import io.kotlintest.shouldBe
```

Comment:
This change should be squashed to 1eefae8004705bb0f3527b1f2254f32726073495, but in exchange the actual test should be added here.

Reply:
Done.